stories/theme/core/Highlighter.css

```diff
@@ -0,0 +1,8 @@
+.Highlighter {
+
+}
+
+.Highlighter_Match {
```

Comment:
`Match` is a modifier so it should be `.Highlighter-match`

Reply:
Gotcha. Changing now